pest/src/lib.rs

```diff
@@ -74,6 +76,8 @@ pub mod prec_climber;
 mod span;
 mod stack;
 mod token;
+#[doc(hidden)]
+pub mod unicode;
```

Comment:
This path should be added to [codecov.yml](https://docs.codecov.io/docs/ignoring-paths).

Reply:
👍 It's been done